src/libinjection_xss.c

```diff
@@ -341,14 +432,20 @@ static attribute_t is_black_attr(const char* s, size_t len)
     }
 
     if (len >= 5) {
-        /* JavaScript on.* */
+
+	/* JavaScript on.* event handlers */
```

Comment:
Not sure what kind of style we're looking for, but indentation on these lines looks odd to me.